### DISCUSSION TITLE
Error writing HYMAP2 routing ensemble outputs

### DISCUSSION THREAD

**Discussion by prakrutkansara** [NASA-LIS/LISF#1576] [General]
Hello,

I get a 'floating overflow' error while writing HYMAP2 routing ensemble outputs on Discover. I am using Generic Ensemble Forecast Reader to read in meteorological forecasts with 7 ensemble members and the output methodology is set to '2d ensemble gridspace'. The NOAH-MP LSM outputs are written correctly but writing routing output gives a 'floating overflow' error. 

I have run this model with retrospective forcing with similar configuration setup except for the Ensemble forecast reader and ensemble options and it works without any error.

Attached are the config files, LIS log and SLURM error and output log files:
[lis_config_ensemble.txt](https://github.com/user-attachments/files/16238816/lis_config_ensemble.txt)
[model_output.tbl.txt](https://github.com/user-attachments/files/16238817/model_output.tbl.txt)
[lislog.txt](https://github.com/user-attachments/files/16238818/lislog.txt)
[slurm_error_log.txt](https://github.com/user-attachments/files/16238819/slurm_error_log.txt)
[slurm_output_log.txt](https://github.com/user-attachments/files/16238820/slurm_output_log.txt)

Any help would be appreciated. Thank you in advance!

**Comment by karsenau** [comment]
A bug fix was provided in Aug-2024 by @prakrutkansara and JHU team and tested by @abheeralisf and FLDAS team.

Will be following up shortly for the code commit, PR and test case for this in the public LIS 7.5 branch

@karsenau @jvgeiger
